params/hooks_libevm.go

```diff
@@ -79,15 +82,18 @@ var PrecompiledContractsGranite = map[common.Address]vm.PrecompiledContract{
 	nativeasset.GenesisContractAddr:    makePrecompile(&nativeasset.DeprecatedContract{}),
 	nativeasset.NativeAssetBalanceAddr: makePrecompile(&nativeasset.DeprecatedContract{}),
 	nativeasset.NativeAssetCallAddr:    makePrecompile(&nativeasset.DeprecatedContract{}),
+	P256VerifyAddress:                  vm.PrecompiledContractsP256Verify[P256VerifyAddress],
```

Comment:
This is risky. If the upstream code changes its address (unlikely but currently still possible) then it will result in the precompile here being nil. I've sent you https://github.com/ava-labs/libevm/pull/222 for review so you can just use `&vm.P256Verify{}` directly once it's merged.